spark/v3.5/spark-runtime/LICENSE

```diff
@@ -403,22 +350,6 @@ License: http://www.apache.org/licenses/LICENSE-2.0
 
 --------------------------------------------------------------------------------
 
-This binary artifact contains findbugs-annotations by Stephen Connolly.
-
-Copyright: 2011-2016 Stephen Connolly, Greg Lucas
-Home page: https://github.com/stephenc/findbugs-annotations
-License: http://www.apache.org/licenses/LICENSE-2.0
-
---------------------------------------------------------------------------------
-
-This binary artifact contains Google j2objc Annotations.
```

Comment:
I don't find j2objc in the spark-runtime jar.

Reply:
We do use this library (from checkstyle <- Guava ) but I believe all the annotations are stripped at compile time. So I don't believe it should be relevant for the binary.

Reply:
Yeah, annotations aren't required at runtime and most of these are annotations Jars. Maybe someone removed them to cut down on unnecessary dependencies and we didn't drop them from LICENSE files?

Reply:
@rdblue I think it's history 😄 That happens 😄